src/Entity/Phone.php

```diff
@@ -144,6 +152,61 @@ public function __construct()
         $this->updatedAt = new DateTimeImmutable();
     }
 
+    /**
+     * hydrate the Phone entity.
+     *
+     * @param null|string $constructor
+     * @param null|string $name
+     * @param null|float  $priceEuro
+     * @param null|string $system
+     * @param null|string $userInterface
+     * @param null|string $processor
+     * @param null|string $ram
+     * @param null|string $capacity
+     * @param null|string $das
+     * @param null|string $batteryCapacity
+     * @param null|bool   $wirelessCharging
+     * @param null|string $weight
+     * @param Size        $size
+     * @param Screen      $screen
+     *
+     * @return self
+     */
+    public function hydrate(
```

Comment:
Codacy found an issue: [The method hydrate has 14 parameters. Consider reducing the number of parameters to less than 10.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6304570)

Reply:
replace parameters by one array paramater